lib/watch-view.js

```diff
@@ -18,11 +20,7 @@ export default class WatchView {
     this.inputEditor.setLineNumberGutterVisible(false);
     this.inputEditor.moveToTop();
 
-    this.resultView = new ResultView();
```

Comment:
We should change the `clearResults` method below since `this.resultView` can be null. (We should flow this up, but it can wait until the PR is merged)

This is the current code:

```js
  clearResults() {
    try {
      this.element.removeChild(this.resultView.element);
      this.resultView.destroy();
    } catch (e) {
      console.error(e);
    }

    this.resultView = new ResultView({ props });
    this.element.appendChild(this.resultView.element);
  }
```

This works better:

```js
clearResults() {
    if (this.resultView != null) {
      this.element.removeChild(this.resultView.element);
      this.resultView.destroy();
    }

    this.resultView = new ResultView({ props });
    this.element.appendChild(this.resultView.element);
  }
```